lib/ims/lti/models/lti_model.rb

```diff
@@ -171,8 +176,12 @@ def self.attributes=(attribs)
       @attributes = attribs
     end
 
-    def self.serialization_options
-      @serialization_options ||= {}
+    def self.serialization_options()
+      if superclass == Object
```

Comment:
It shouldn't make a difference.  The idea was to stop once I got to the LtiModel class, so the check could have been to check if the class name was LtiModel. But every subclass below it will have the "serilization_options" method since it inherits from here.